Update Mannul Chapter 35 (#235)
eisop/checker-framework: docs/manual/creating-a-checker.tex

% \refqualclass{framework/qual}{SubtypeOf} and
% \sunjavadocanno{java.base/java/lang/annotation/Target.html}{Target}.)

> The qualifiers should be in a \<qual> subpackage of the type checker subpackage -

Reviewer: ```suggestion
The qualifiers should be in a \<qual> subpackage of the type checker package ---
```

Author: Thanks!

> that is the checker reflectively looks and the only thing that matters for that is the package name.

Reviewer: ```suggestion
by default, the checker reflectively determines the qualifiers and the only thing that matters for that is the package name.
```

Author: Thanks!

As mentioned in {creating-parts-of-a-checker}, the files for that \<qual> package can be
either in a subdirectory or in a separate gradle module, depending on whether

> they should be distributable separately. The Checker Framework automatically treats any annotation that

Reviewer: ```suggestion
they should be distributable separately.
The Checker Framework automatically treats any annotation that
```

Author: Thanks!

% \sunjavadocanno{java.base/java/lang/annotation/Target.html}{Target}.)

The annotations should be placed within a directory called \<qual>, and

> \<qual> may not be in the same directory as your checker's source file.

Author: can put in..., and can be placed a sep modu....

% \sunjavadocanno{java.base/java/lang/annotation/Target.html}{Target}.)

> The annotations should be placed within a directory called \<qual>, and

Reviewer: Maybe split this up into two: the qualifiers should be in a `qual` sub-package of the type checker subpackage - that's were the checker reflectively looks and the only thing that matters for that is the package name.
As mentioned in `{creating-parts-of-a-checker}` the files for that `qual` package can be either in a subdirectory or in a separate gradle module, depending on whether they should be distributable separately. Briefly repeat that point.

Author: Thanks, updated.

programmer didn't write a type qualifier explicitly.

\subsectionAndLabel{Completeness of the qualifier hierarchy}{creating-bottom-and-top-qualifier}

> When you define a type system, its type hierarchy must be a

Author: its type hierarchy

Author: find others.
